app/docs/0.13.x/proxy.md

```diff
@@ -845,51 +890,53 @@ the client:
   response.
 
 Once the headers are sent to the client, Kong will start executing
-registered plugins for that API that implement the `body_filter` hook. This
-hook may be called multiple times, due to the streaming nature of Nginx itself.
-Each chunk of the upstream response that is successfully processed by such
-`body_filter` hooks is sent back to the client. You can find more informations
-about the `body_filter` hook in the [Plugin development
-guide][plugin-development-guide].
+registered plugins for the Route and/or Service that implement the
+`body_filter` hook. This hook may be called multiple times, due to the
+streaming nature of Nginx. Each chunk of the upstream response that is
+successfully processed by such `body_filter` hooks is sent back to the client.
+You can find more informations about the `body_filter` hook in the [Plugin
+development guide][plugin-development-guide].
 
 [Back to TOC](#table-of-contents)
 
-## Configuring a fallback API
+## Configuring a fallback Route
 
 As a practical use-case and example of the flexibility offered by Kong's
-proxying capabilities, let's try to implement a "fallback API", so that in
-order to avoid Kong responding with an HTTP `404`, "API not found", we can
+proxying capabilities, let's try to implement a "fallback Route", so that in
+order to avoid Kong responding with an HTTP `404`, "no route found", we can
 catch such requests and proxy them to a special upstream service of yours, or
```

Comment:
Since you have removed other "of yours" previously, you may want to remove the one in "a special upstream service of yours" as well